src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,67 @@
+package duke.command;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+
+import duke.DukeException;
+import duke.usertask.TaskList;
+
+/**
+ * ListCommand handles outputting the entire list of tasks that a user has at the current point in time.
+ * It has an optional argument field <code>dateFilter</code>, which will filter the list of tasks to output
+ * based on whether they 1. have a <code>dateTime</code> field, and 2. the date of the field is equal to the specified
+ * dateFilter supplied by the user. <b>Note:</b> time is irrelevant here, and filter works solely based on date.
+ */
```

Comment:
I really like the level of detail you have for headers! It is consistent in almost every class I have seen!

Reply:
Thank you for your kind words and feedback!